src/cond.d

```diff
@@ -25,6 +25,20 @@ import ddmd.root.outbuffer;
 import ddmd.tokens;
 import ddmd.visitor;
 
+private __gshared Identifier idUnitTest;
```

Comment:
If we keep using `Token.toChars(TOK)` static function, moving it might cause circular dependency between `id.d` / `identifier.d` and `tokens.d` modules, and generated `id.c` and its generator `idgen.d` would also have to be tweaked.

That's too big change compared with the original purpose of this PR, so I'd like to keep this minimal change.